lib/monitoring/aws-dynamo/DynamoTableGlobalSecondaryIndexMetricFactory.ts

```diff
@@ -48,24 +50,28 @@ export class DynamoTableGlobalSecondaryIndexMetricFactory {
   }
 
   metricConsumedReadCapacityUnits() {
-    return this.metricFactory.createMetric(
-      "ConsumedReadCapacityUnits",
-      MetricStatistic.SUM,
-      "Consumed",
-      this.dimensionsMap,
-      undefined,
-      DynamoDbNamespace
+    return this.metricFactory.createMetricMath(
+      "consumed_rcu_sum/PERIOD(consumed_rcu_sum)",
+      {
+        consumed_rcu_sum: this.table.metricConsumedReadCapacityUnits({
+          statistic: MetricStatistic.SUM,
+          dimensionsMap: this.dimensionsMap,
+        }),
+      },
+      "Consumed"
```

Comment:
Why can't we just use the metric provided by the table construct?

Reply:
https://docs.aws.amazon.com/cdk/api/v2/docs/aws-cdk-lib.aws_dynamodb.Table.html#metricwbrconsumedwbrreadwbrcapacitywbrunitsprops

> By default, the metric will be calculated as a sum over a period of 5 minutes. You can customize this by using the `statistic` and `period` properties.

We'd want to adapt this to whatever the relevant period is similar to what's being done for the non-GSI DDB metrics as mentioned in #362.